scripts/test/diff_dashboard_states.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Requirements:
+# - Imagemagick installed with PNG support locally
+
+set -e -o pipefail
+
+BRANCH1=$(git rev-parse --abbrev-ref HEAD)
+BRANCH2=$1
+
+shift 1
+
+OUT1="output/dashboard_states_A"
```

Comment:
Instead of `A` and `B` can we name them `current` and `other`, or maybe use the branch names themselves, mangled to remove the `/`?